tests/metabox/class-metabox-collapsibles-section-test.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Yoast\WP\Free\Tests\Admin\Metabox;
+
+use Yoast\WP\Free\Tests\TestCase;
+use \WPSEO_Metabox_Collapsibles_Section;
+use \WPSEO_Metabox_Collapsible;
+use \WPSEO_Abstract_Sectioned_Metabox_Tab;
+use Brain\Monkey;
+
+/**
+ * Unit Test Class.
+ */
+class WPSEO_Metabox_Collapsibles_Section_Test extends TestCase {
+
+	/**
+	 * Tests the output of \WPSEO_Metabox_Collapsibles_Section::display_content.
+	 *
+	 * @covers WPSEO_Metabox_Collapsibles_Section::__construct
```

Comment:
FQCNs, same as comment above.

Reply:
This has to be the full FQN, correct, but it doesn't need to start with a `\`.